### ISSUE TITLE
Add Sermon-Online Repository 

### ISSUE THREAD

**Issue by Berovowilare** [mdbergmann/Eloquent#77]
Hallo Manfred

Ich probierte das Sermon-Online Repository hinzuzufügen, wie es von https://luki.org/2020/11/quelloffene-bibelsoftware-2-freie-und-offene-formate-fuer-theologische-inhalte-und-bibeln/ erwähnt wird. Leider funktionierte es nicht. 

Magst du schauen weshalb?
<img width="688" alt="Bildschirmfoto 2023-05-10 um 19 30 04" src="https://github.com/mdbergmann/Eloquent/assets/36891575/c709d8da-8e60-4240-8f32-2d9fa3a54550">

Evtl. sind auch noch andere Module verfügbar als nur Bibeln auf Deutsch.

Danke

**Comment by mdbergmann** [comment]
Also, so wie ich es sehe müßte 'Host' ohne schema angegeben werden, also nur "ftp.sermon-online.com", so wie auch bei den anderen Installationsquellen.

Zum Anderen, wie in dem Link geschildert muss der Pfad sein: "/german/Bible/Sword"

Man kommt dann mit einem FTP client auf den Server, und da sind auch Sword module, allerdings fehlt eine Datei: "mods.d.tar.gz". Auf die Existenz dieser Datei wird geprüft damit eine Installationsquelle "valide" ist.

Laut dem Wiki Artikel scheint diese Datei nicht zwingend erforderlich zu sein aber dennoch erwarten die meisten Sword Programme diese.
https://wiki.crosswire.org/Creating_and_Maintaining_a_Module_Repository

Vielleicht kannst du den Server Admin mal anhauen ob er die nicht erstellen kann.

**Comment by Berovowilare** [comment]
Nein: Ich schrieb sie an. Sogar mit einem Brief an die Postadresse, aber es kam nie was zurück.
